storage/types.go

```diff
@@ -168,7 +168,7 @@ func NewNodeIDFromPrefix(prefix []byte, depth int, index int64, subDepth, totalD
 // newNodeIDFromBigIntOld returns a NodeID of a big.Int with no prefix.
 // index contains the path's least significant bits.
 // depth indicates the number of bits from the most significant bit to treat as part of the path.
-func newNodeIDFromBigIntOld(depth int, index *big.Int, totalDepth int) NodeID {
+func newNodeIDFromBigIntOld(depth int, index *big.Int, totalDepth int) *NodeID {
```

Comment:
What's the status of the old / new bigint constructors? Do we need both of them? (removal not needed for this PR)

Reply:
I'll remove it in a new PR